Trustroots/trustroots
# Add public feedback UI

#### Proposed Changes

* add third tab to the create reference flow to add optional public feedback

![tr-feedback](https://user-images.githubusercontent.com/31616/79689574-0885b480-8256-11ea-9c97-0dcdc1a6f570.png)


#### Testing Instructions

* `git switch add/feedback-public-ui`
* `npm start`
* find a user on the map
* view their profile
* click "create a reference" near the top
* go through the 3 steps
* add some public feedback text
* see that it submits successfully
* (it won't be displayed on their profile, that is being done in a future PR)
* ... try the whole thing again, but _don't_ leave a public reference
* it should still work (as it's optional)

I would particularly like feedback on the text on the page, it should maybe be clearer that it is optional, and could have some better guidance for the user. As I understand it, @Akronix was particularly keen to have it that the user is writing content that gives a more complete picture of the user, to complement their profile, so not really writing about the visit/stay/meet itself, but about the person.

Some optimizations/tweaks I could add if people are interested:
- have a checkbox to first tick, to confirm they want to write something, before showing the text box
- ... maybe that's it

There are a few thoughts from @simison on the old PR: https://github.com/Trustroots/trustroots/pull/1384#issuecomment-613518304

Part of https://github.com/Trustroots/trustroots/issues/718
